src/app/profile/yourGiving/stopStartRecurringGifts/step0/stopStartStep0.tpl.html

```diff
@@ -0,0 +1,80 @@
+<div class="modal-content give-modal-content">
+  <div class="modal-header">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-xs-12">
+          <div class="border-bottom-small">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">×</span></button>
+            <h3 translate>What would you like to do?</h3>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-body">
+    <div class="container-fluid" tabindex="-1" role="dialog">
```

Comment:
Do we need `tabindex="-1" role="dialog"`? I've been removing them. I think `$uibModal` sets these on a div that is higher up the DOM. Idk why the UX guys have it only on the body. I could be wrong...